services/addons/images/count_metric/README.md

```diff
@@ -38,44 +38,8 @@ To run this service, the following environment variables must be set:
 
 <b>SMTP_EMAIL_RECIPIENTS:</b> Recipient emails, delimited by ','.
 
-## Kafka JPO-ODE Message Counter
+## Daily Counter (MongoDB)
 
-### NOTE: The following is now deprecated. It is recommended to use the emailer feature of this addon service with the mongoDB deployment from the [jpo-conflictmonitor](https://github.com/usdot-jpo-ode/jpo-conflictmonitor/blob/develop/docker-compose.yml) repository with the jpo-ode. The old feature will be removed in the next release (Summer 2024).
+The daily counter is a feature that aggregates JPO-ODE mongoDB message type counts for BSM, TIM, Map, SPaT, SRM and SSM and inserts them into a new collection in mongoDB. This new collection is named "CVCounts". This new collection is useful for the CV Manager to query the message counts in a performant manner. This script runs on a cron every 24 hours.
```

Comment:
Does this need to be updated to include PSMs as a supported message type?

Reply:
Yes, good catch! Updated.